client/schema/LBSettings.schema.json

```diff
@@ -76,7 +76,11 @@
             "type": "array",
             "items": { "type": "string", "pattern": "^(\\d{3}|all)$" }
           },
-          "CacheTime": { "$ref": "#/definitions/nonEmptyString" }
+          "CacheTime": { "$ref": "#/definitions/nonEmptyString" },
+          "RestrictTo": {
+              "type": "array",
+              "items": { "type": "string", "pattern": "^([0-9]{1,3}\\.){3}[0-9]{1,3}(\\/[0-9]{1,2})?$" }
```

Comment:
Validation of this regex shows an unescaped character `\\/` (`/`) and is causing an error. Could you make give me an example of what you would like this to match and make sure it passes regex validation using `Javascript` reex engine here: https://regex101.com/